### ISSUE TITLE
Typing bug when typing a commit message in Chinese

### ISSUE THREAD

**Issue by quancs** [microsoft/vscode#115249]
Issue Type: <b>Bug</b>

# Step 1
Create a file, and create a git repo

# Step 2
Open this file in vscode and type commit message in Chinese. Very often, when you are typing commit messages, your character selection bar will get stuck, and you cannot type anymore. You can only solve the problem by restarting vscode.

# VS Code info

VS Code version: Code 1.52.1 (ea3859d4ba2f3e577a159bc91e3074c5d85c0523, 2020-12-16T16:34:46.910Z)
OS version: Windows_NT x64 [10.0.19042]

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9700 CPU @ 3.00GHz (8 x 3000)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.82GB (7.20GB free)|
|Process Argv|--crash-reporter-id 06638104-59e4-4ad0-8f37-59ea6ec2ff2c|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (14)</summary>

Extension|Author (truncated)|Version
---|---|---
code-runner|for|0.11.2
gc-excelviewer|Gra|3.0.40
latex-workshop|Jam|8.15.0
vscode-language-pack-zh-hans|MS-|1.52.2
anaconda-extension-pack|ms-|1.0.1
python|ms-|2021.1.502429796
jupyter|ms-|2020.12.414227025
remote-wsl|ms-|0.52.0
vsliveshare|ms-|1.0.3577
autodocstring|njp|0.5.4
vscode-yaml|red|0.14.0
LiveServer|rit|5.6.1
pytorch-snippets|SBS|1.0.2
ZwLisp|Zws|1.0.8

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
openlogontheside:30221877
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30211401
wsl2prompt:30224612
pythonvsdeb440:30248342
unusedprompt:30224610
folderexplorercf:30224615
openfilemenucf:30224648
pythonvsded773:30248341
pythonvspyt600cf:30247032

```

</details>

**Comment by tamuratak** [comment]
Duplicate of #112621

**Comment by eamodio** [comment]
FYI, this has been fixed in 1.53
